dbms/src/Flash/Planner/Plans/PhysicalAggregationBuild.cpp

```diff
@@ -77,6 +77,7 @@ EventPtr PhysicalAggregationBuild::doSinkComplete(PipelineExecutorContext & exec
         aggregate_context.reset();
         return nullptr;
     }
+    aggregate_context->getAggSpillContext()->finishSpillableStage();
```

Comment:
Maybe moving to L74 like `ParallelAggBlockInputStream::onFinish`?

Reply:
Done